src/v2/view-builder/views/phone/ChallengeAuthenticatorPhoneView.js

```diff
@@ -57,12 +57,19 @@ const Body = BaseForm.extend(Object.assign(
         : loc('mfa.calling', 'login');
       const enterCodeText = loc('oie.phone.verify.enterCodeText', 'login');
       const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
+      let nicknameText = '';
+      if (this.model.get('nickname')) {
+        nicknameText = '(' + this.model.escape('nickname') + ')';
+      }
+      if (nicknameText.length > 20) {
+        nicknameText = nicknameText.substring(0, 20) + '...';
+      }
 
       const strongClass = this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
         'strong no-translate' : '';
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">
-        ${sendText}&nbsp;<span class='${strongClass}'>${this.model.escape('phoneNumber')}.</span>
+        ${sendText}<br/><span class='${strongClass}'>${this.model.escape('phoneNumber')}<br/>.${nicknameText}</span>
```

Comment:
Any reason why we are adding a line break here but not in the other view? Is this necessary? Has anyone from Design reviewed this change?

cc: @alexdahl-okta

Reply:
Line breaks were added based on informal feedback from Alex and Joel. However, I don't see them show up at all even after re-deploying my server. Is there any override that could be preventing it? Thanks

Reply:
Reviewed feedback from Joel and Alex with Glen. Resolving this thread as removing the line breaks seems to address it.
cc: @glenfannin-okta

Reply:
@diptidesai-okta Can we see screenshots with the line break working? I want to make sure the nickname line wraps normally.